File: subscriptions/models/subscriptions.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const Subscriptions = new Schema({
    id             : {type: Number, required: false},
    event_id       : {type: Number, required: true},
    user_id        : {type: Number, required: true},
    subscribed_at  : {type: Date, required: false},
    unsubscribed_at: {type: Date, required: false},
    checkin_at     : {type: Date, required: false},
}, {
    versionKey: false,
});

module.exports = mongoose.model('Subscriptions', Subscriptions)

File: checkin/models/subscriptions.py
from mongoengine import Document, IntField, DateTimeField

class Subscriptions(Document):
    event_id = IntField(required=False)
    user_id = IntField(required=False)
    subscribed_at = DateTimeField(required=False)
    unsubscribed_at = DateTimeField(required=False)
    checkin_at = DateTimeField(required=False)
